### ISSUE TITLE
No Brier Score Loss Function

### ISSUE THREAD

**Issue by tambulkar** [tensorflow/tensorflow#31951]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 1.14.0
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
https://en.wikipedia.org/wiki/Brier_score

**Will this change the current api? How?**
Add support for calculating Brier Score

**Who will benefit with this feature?**
Anyone who wants to utilize Brier Score

**Any Other info.**

**Comment by tambulkar** [comment]
https://github.com/tensorflow/tensorflow/pull/31950
